packages/opencensus-nodejs/test/instrumentation/node_modules/@opencensus/instrumentation-simple-module/simple-module.ts

```diff
@@ -20,18 +20,36 @@ import * as shimmer from 'shimmer';
 
 
 export class SimpleModulePlugin extends classes.BasePlugin {
+  protected internalFileList: types.PluginInternalFiles = {
```

Comment:
This module doesn't seem very simple anymore. Can you have a separate instrumentation fixture for something that requires an internal module to be patched.

Reply:
A new one was created.